lib/debug/server_cdp.rb

```diff
@@ -229,6 +234,8 @@ def process
           @q_msg << req
         end
       end
+    rescue Restart
+      @q_msg << 'open chrome'
```

Comment:
sorry what's happen on disconnect?

Reply:
When the connection is disconnected, debugger restarts as follows:

```shell
% exe/rdbg target2.rb --open=chrome --port 12345
DEBUGGER: Debugger can attach via TCP/IP (127.0.0.1:12345)
DEBUGGER: With Chrome browser, type the following URL in the address-bar:

   devtools://devtools/bundled/inspector.html?ws=127.0.0.1:12345

DEBUGGER: wait for debugger connection...
DEBUGGER: Connected.
# A tab is closed here
DEBUGGER: Disconnected.
DEBUGGER: wait for debugger connection...
DEBUGGER: Debugger can attach via TCP/IP (127.0.0.1:12345)
DEBUGGER: With Chrome browser, type the following URL in the address-bar:

   devtools://devtools/bundled/inspector.html?ws=127.0.0.1:12345
```

Reply:
Is this the wrong way?